app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -99,7 +99,7 @@ def authenticated_with_all_providers?
       end
 
       def user_following_tags?
-        user.cached_followed_tag_names.count > 1
```

Comment:
I'm not sure why we used `> 1` here instead of `>= 1` but `.any?` seems to actually make the most sense.